src/views/LotteryV2/components/BuyTicketsModal/TicketInput.tsx

```diff
@@ -207,6 +220,7 @@ const TicketContaier: React.FC<{
           onFocus={onFocusHandler}
           onBlur={onBlurHandler}
           onPaste={onPasteHandler}
+          inputMode="numeric"
```

Comment:
Pattern validation is needed too?

Reply:
Not really, pattern validation is done above in `onChangeHandler` (it has quite convoluted way of checking whether input is allowed or not, other things are ignored)

Reply:
I can add it just for double protection though 😄